draft-ietf-webtrans-http3.md

```diff
@@ -493,15 +506,15 @@ stream or a datagram before receiving the CONNECT response headers from the
 server.
 
 To handle this case, WebTransport endpoints SHOULD buffer streams and datagrams
-until those can be associated with an established session.  To avoid resource
-exhaustion, the endpoints MUST limit the number of buffered streams and
-datagrams.  When the number of buffered streams is exceeded, a stream SHALL be
-closed by sending a RESET_STREAM and/or STOP_SENDING with the
-`WT_BUFFERED_STREAM_REJECTED` error code.  When the number of
-buffered datagrams is exceeded, a datagram SHALL be dropped.  It is up to an
+until they can be associated with an established session.  To avoid resource
+exhaustion, endpoints MUST limit the number of buffered streams and datagrams.
+When the number of buffered streams is exceeded, a stream SHALL be closed by
```

Comment:
Are we killing all the SHALL's here?

Reply:
Not 100% so far, but working in that direction